data/assets.go

```diff
@@ -1,4 +1,6 @@
+//go:build !release
 // +build !release
```

Comment:
Should we remove the `+build` directives, too, and force the move to go1.17 by changing https://github.com/openshift/installer/blob/cd170249fb5d7ce041ee35aae1486872cb076ef6/hack/build.sh#L8?

Reply:
I think that's a bit premature.  For now I think it's sufficient to make sure installer *can* build with 1.17.